frontend_v2/src/app/components/challenge/challengesettings/challengesettings.component.html

```diff
@@ -72,6 +71,32 @@ <h5 class="fw-light">Challenge Settings</h5>
       </div>
     </div>
 
+    <!-- Evaluations and Annoatations -->
+    <div class="row settings-section">
+      <div class="col-lg-3 col-md-4 col-sm-3 col-xs-3">
+        <span class="fw-light fs-18">Evaluation and Annoatations</span>
```

Comment:
@gautamjajoo can we please fix the typo here? `Annoatations` -> `annotations`

Reply:
So sorry for the typo!